plugin/evm/atomic/atomic_backend.go

```diff
@@ -22,6 +23,12 @@ import (
 
 var _ AtomicBackend = &atomicBackend{}
 
+var (
+	// Prefixes for atomic trie
+	atomicTrieDBPrefix     = []byte("atomicTrieDB")
+	atomicTrieMetaDBPrefix = []byte("atomicTrieMetaDB")
```

Comment:
nit unneeded comment

```suggestion
	atomicTrieDBPrefix     = []byte("atomicTrieDB")
	atomicTrieMetaDBPrefix = []byte("atomicTrieMetaDB")
```